src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -256,6 +257,26 @@ protected static bool ValidateManifestsInTempDir(Manifests manifests)
             return result;
         }
 
+        /// <summary>
+        /// Displays the appropriate warning messages for installers with detected architecture mismatches.
+        /// </summary>
+        /// <param name="detectedArchs">List of DetectedArch objects that represent each installers detected architectures.</param>
+        protected static void DisplayMismatchedArchitectures(List<PackageParser.DetectedArch> detectedArchs)
+        {
+            var mismatchedArchInstallers = detectedArchs.Where(i => i.UrlArch != i.BinaryArch).ToList();
```

Comment:
I'd say this should be where i.UrlArch.HasValue && i.UrlArch != i.BinaryArch

Reply:
Changed to suggested